plio/tests.py

```diff
@@ -135,3 +135,106 @@ def setUp(self):
     def test_for_question(self):
         # write API calls here
         self.assertTrue(True)
+
+
+class ImageTestCase(PlioTestCase):
+    """Tests the Image CRUD functionality."""
+
+    def setUp(self):
+        # seed some plios
+        super().setUp()
+
+        # seed some items and questions
+        self.plio_1 = Plio.objects.filter(name="Plio 1").first()
+
+        self.item_1_plio_1 = Item.objects.create(
+            plio=self.plio_1, type="question", time=1
+        )
+        self.item_2_plio_1 = Item.objects.create(
+            plio=self.plio_1, type="question", time=2
+        )
+
+        self.question_1_plio_1 = Question.objects.create(item=self.item_1_plio_1)
+        self.question_2_plio_1 = Question.objects.create(item=self.item_2_plio_1)
+
+    def test_user_can_attach_images_to_their_question(self):
+        """
+        Tests whether a user can link images to the questions that were created by themselves
+        """
+        # upload a test image and retrieve the id
+        with open("plio/static/plio/test_image.jpeg", "rb") as img:
+            response = self.client.post(reverse("images-list"), {"url": img})
+        uploaded_image_id = response.json()["id"]
+
+        # update the question with the newly uploaded image
+        response = self.client.put(
+            reverse("questions-detail", args=[self.question_1_plio_1.id]),
+            {"item": self.item_1_plio_1.id, "image": uploaded_image_id},
+        )
+        # the user should be able to update the question
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_cannot_attach_image_to_other_user_question(self):
+        """
+        Tests that a user should NOT be able to link images to the questions that were created by some other user
+        """
+        # create a new user
+        new_user = User.objects.create(mobile="+919988776654")
```

Comment:
Vaibhav used the number `+919876543210` as dummy - best to use the same everywhere?